logstash-core-event-java/src/main/java/com/logstash/Rubyfier.java

```diff
@@ -45,8 +50,7 @@ private static RubyArray deepList(Ruby runtime, final List list) {
         final RubyArray array = runtime.newArray(length);
 
         for (Object item : list) {
-            // use deepOnly because RubyArray.add already calls JavaUtil.convertJavaToUsableRubyObject on item
-            array.add(deepOnly(runtime, item));
+            array.add(deep(runtime, item));
```

Comment:
I guess its stored in git so if we remove it now we can get it back if its needed again.

Will remove.